sno/crs_util.py

```diff
@@ -50,3 +50,23 @@ def get_identifier_int(crs):
         return int(auth_code)
     # Stable code that fits easily in an int32 and won't collide with EPSG codes.
     return (hash(crs.ExportToWkt()) & 0xFFFFFFF) + 1000000
```

Comment:
`hash()` in Python isn't stable...

Reply:
Fixed.
(This code is not often exercised, it's only used if there is a custom CRS without a code)